CONTRIBUTING.md

```diff
@@ -69,23 +69,30 @@ To run the test suite on your machine you have multiple options:
 
 - **Full test suite with tox:** Run the full suite across different Python versions with
   
-  ```
+  ```bash
   tox
   ```
-- **Single environment with tox:** Run tests via [tox](https://probabilistic-numerics.github.io/probnum/development/contributing.html#tox) for a single Python environment. Example for Python 3.6:
+- **Single environment with tox:** Run tests for a single Python environment. Example for Python 3.6:
   
-  ```
+  ```bash
   tox -e py36
   ```
 - **pytest:** Run tests directly in your local environment by calling
   
-  ```
+  ```bash
   pytest
   ```
 
 ## Documentation
 
-[Documentation](https://probabilistic-numerics.github.io/probnum/modules.html) is automatically built using [Sphinx](https://www.sphinx-doc.org/en/master/) and [Travis](https://travis-ci.org/probabilistic-numerics/probnum).
+ProbNum's documentation is automatically built using [Sphinx](https://www.sphinx-doc.org/en/master/) and 
+[Travis](https://travis-ci.org/probabilistic-numerics/probnum) and hosted by [ReadTheDocs](https://readthedocs.org/projects/probnum/).
+You can access the documentation for the default version and the development branch here:
+
+[![default docs](https://img.shields.io/readthedocs/probnum.svg?logo=read%20the%20docs&logoColor=white&label=Docs)](https://probnum.readthedocs.io)
```

Comment:
I like "stable" and "latest" to refer to the two different states, but "default" and "development" is of course also fine for me. I don't really know how others call those.

Reply:
In the latest and stable language latest would be `master` and `stable` a release. Is this what we want at this point? This would also not expose the developer docs. My suggestion would be to have `stable`:release, `latest`:master, `dev`:development

Reply:
Then I might have misunderstood the meaning of stable/latest. The cases in which I saw that naming seemed to follow a workflow in which they work directly on master, without using a development branch. Therefore I assumed that the mapping to our workflow would be to use stable/latest for master/development, but this might very well be completely false. 

I like your proposal of using:
- stable:release
- latest:master
- dev:development